tools/scripts/seed/seedAuthUser.js

```diff
@@ -22,18 +22,15 @@ function handleError(err, client) {
   }
 }
 
-MongoClient.connect(MONGOHQ_URL, { useNewUrlParser: true }, function(
-  err,
-  client
-) {
+MongoClient.connect(MONGOHQ_URL, { useNewUrlParser: true }, (err, client) => {
   handleError(err, client);
 
   log('Connected successfully to mongo');
 
   const db = client.db('freecodecamp');
   const user = db.collection('user');
 
-  user.deleteOne({ _id: ObjectId('5bd30e0f1caf6ac3ddddddb5') }, err => {
+  user.deleteMany(err => {
```

Comment:
```suggestion
  user.deleteMany(
    {
      _id: {
        $in: [
          ObjectId('5bd30e0f1caf6ac3ddddddb5'),
          ObjectId('5bd30e0f1caf6ac3ddddddb9')
        ]
      }
    },
    err => {
```
Let's err on the side of caution and just drop the records we know about.